pom.xml

```diff
@@ -231,7 +231,7 @@
     <dependency>
         <groupId>io.jenkins.plugins</groupId>
         <artifactId>jaxb</artifactId>
-        <version>2.3.0</version>
+        <version>2.3.0.1</version>
```

Comment:
As per https://github.com/jenkinsci/artifact-manager-s3-plugin/pull/87#discussion_r269672000 I suppose. But is the `test` scope really right? I thought the idea was that we need to declare an actual dep on the `jaxb` plugin if we use it.

Reply:
Ah, RTFM

> It is currently not recommended to depend on it from your plugin.
> It is designed to be automatically added as a dependency when running Jenkins on a Java 11 runtime, so that plugins using JAXB API do not fail on Java 11 (given JAXB libraries were removed from the JDK in Java 9).